engine-tests/src/test/java/org/terasology/engine/module/ModuleDownloadListGeneratorTest.java

```diff
@@ -1,131 +1,131 @@
-        ModuleDownloadListGenerator listGenerator = new ModuleDownloadListGenerator(localRegistry, resolver);
-        assertEquals(Collections.singleton(moduleV2), buildList(listGenerator));
-    }
-
-    @Test
-    public void testMultipleModulesPartialUpdate() throws DependencyResolutionFailedException {
-        Module moduleAV1 = buildSimpleModule("myModuleA", "1.0.0");
-        Module moduleBV1 = buildSimpleModule("myModuleB", "1.0.0");
-        Module moduleBV2 = buildSimpleModule("myModuleB", "2.0.0");
-        ModuleRegistry localRegistry = buildRegistry("1.0.0", moduleAV1, moduleBV1);
-        DependencyResolver resolver = mockResolver(true, moduleBV1, moduleBV2, buildEngineModule("1.0.0"));
-        ModuleDownloadListGenerator listGenerator = new ModuleDownloadListGenerator(localRegistry, resolver);
-        assertEquals(Collections.singleton(moduleBV2), buildList(listGenerator));
-    }
-
-    private DependencyResolver mockResolver(boolean success, Module... resolutionResult) {
-        DependencyResolver result = mock(DependencyResolver.class);
-        DependencyResolver.ResolutionBuilder builder = mock(DependencyResolver.ResolutionBuilder.class);
-        when(builder.requireVersion(any(), any())).thenReturn(builder);
-        when(builder.requireAll(any(Name[].class))).thenReturn(builder);
-        when(builder.build()).thenReturn(new ResolutionResult(success, new HashSet<>(Arrays.asList(resolutionResult))));
-        when(result.builder()).thenReturn(builder);
-        return result;
-    }
-
-    private Set<Module> buildList(ModuleDownloadListGenerator generatorWithMockedResolver) throws DependencyResolutionFailedException {
-        return generatorWithMockedResolver.getAllModulesToDownloadFor();
-    }
-
-    private Module buildSimpleModule(String id, String version) {
-        ModuleMetadata metadata = new ModuleMetadata();
-        metadata.setId(new Name(id));
-        if (version != null) {
-            metadata.setVersion(new Version(version));
-        }
-        return new BaseModule(Collections.emptyList(), metadata) {
-            @Override
-            public ImmutableList<URL> getClasspaths() {
-                return null;
-            }
-
-            @Override
-            public boolean isOnClasspath() {
-                return false;
-            }
-
-            @Override
-            public boolean isCodeModule() {
-                return false;
-            }
-        };
-    }
-
-    private Module buildEngineModule(String version) {
-        return buildSimpleModule(TerasologyConstants.ENGINE_MODULE.toString(), version);
-    }
-
-    private ModuleRegistry buildRegistry(String engineVersion, Module... modules) {
-        ModuleRegistry result = new TableModuleRegistry();
-        result.add(buildEngineModule(engineVersion));
-        result.addAll(Arrays.asList(modules));
-        return result;
-    }
-}
+///*
```

Comment:
There must be a better way of dealing with this than commenting out the entire file. Isn't there some mechanism for ignoring tests?

Reply:
> There must be a better way of dealing with this than commenting out the entire file. Isn't there some mechanism for ignoring tests?

It is draft PR. :P

Compile time problems. 

This test and ModuleListDownloader will be reimplemented for gestalt v7